src/System.Net.WebClient/tests/WebClientTest.cs

```diff
@@ -399,8 +399,8 @@ public static void UseDefaultCredentials_Roundtrips()
             await Assert.ThrowsAsync<WebException>(() => wc.DownloadStringTaskAsync(System.Net.Test.Common.Configuration.Http.RemoteEchoServer));
         }
 
-        [ActiveIssue(18680)]
-        [OuterLoop("Networking test talking to remote server: issue #11345")]
+        //[ActiveIssue(18680)]
+        //[OuterLoop("Networking test talking to remote server: issue #11345")]
```

Comment:
If you are fixing the bug, then you should delete the ActiveIssue.  But you need to keep the Outerloop one.  You shouldn't leave them as commented out.